addons/vehicles/CfgVehicles/Viper.inc

```diff
@@ -0,0 +1,191 @@
+
+class SWS_V_MBT_M700_Viper : I_MBT_03_cannon_F {
+    ITEM_META(2);
+    displayName = QNAME(M700 Viper);
+    faction = "SWS_UNSC";
+    side = 1;
+    crew = "OPTRE_UNSC_Marine_Soldier_Crewman";
+    typicalCargo[] = { "OPTRE_UNSC_Marine_Soldier" };
+    editorPreview = QPATHTOF(data\viper\editor_preview.jpg);
+    
+    radarType=2;
+    reportRemoteTargets=1;
+    reportOwnPosition=1;
+    receiveRemoteTargets=1;
+    class Components: Components {
+        class SensorsManagerComponent {
+            class Components {
+                class DataLinkSensorComponent: SensorTemplateDataLink { };
+                class PassiveRadarSensorComponent: SensorTemplatePassiveRadar {
+                    class AirTarget {
+                        minRange=8000;
+                        maxRange=8000;
+                        objectDistanceLimitCoef=-1;
+                        viewDistanceLimitCoef=-1;
+                    };
+                    class GroundTarget {
+                        minRange=8000;
+                        maxRange=8000;
+                        objectDistanceLimitCoef=-1;
+                        viewDistanceLimitCoef=-1;
+                    };
+                };
+            };
+        };
+    };
+
+    class Turrets: Turrets {
+        class MainTurret: MainTurret {
+            class Turrets: Turrets {
+                class CommanderOptics: CommanderOptics {
+                    weapons[] = {
+                        "HMG_127_APC",
+                        "Laserdesignator_mounted",
+                        "SmokeLauncher"
+                    };
+                    magazines[] = {
+                        "200Rnd_127x99_mag_Tracer_Yellow",
+                        "200Rnd_127x99_mag_Tracer_Yellow",
+                        "200Rnd_127x99_mag_Tracer_Yellow",
+                        "SmokeLauncherMag",
+                        "SmokeLauncherMag",
+                        "SmokeLauncherMag",
+                        "SmokeLauncherMag",
+                        "SmokeLauncherMag",
+                        "Laserbatteries"
+                    };
+                    class Components {
+                        class VehicleSystemsDisplayManagerComponentLeft: VehicleSystemsTemplateLeftCommander {
+                            class Components: components {
+                                class SensorDisplay {
+                                    componentType="SensorsDisplayComponent";
+                                    range[]={4000,2000,1000,8000};
+                                    resource="RscCustomInfoSensors";
+                                };
+                            };
+                        };
+                        class VehicleSystemsDisplayManagerComponentRight: VehicleSystemsTemplateRightCommander {
+                            defaultDisplay="SensorDisplay";
+                            class Components: components {
```

Comment:
```suggestion
                            class Components: Components {
```

Reply:
make sure you're using the latest hemtt binary, because it will warn about this.